### PR TITLE
ci(release): publish hydro_cli

### PR DESCRIPTION
**Pull request by MingweiSamuel**
Will bump versions for python pypi publish

### CODE REVIEW

**Review by shadaj — APPROVED** [review]

File: .github/workflows/build-cli.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
 jobs:
   pre_job:
     runs-on: ubuntu-latest
+    if: "github.event_name != 'release' || startsWith(github.ref, 'refs/tags/hydro_cli-')"
```
Comment (shadaj) [inline .github/workflows/build-cli.yml:17]: should this be `&&`?

Comment (shadaj) [reply .github/workflows/build-cli.yml:17]: and then we can eliminate all conditionals

Comment (MingweiSamuel) [reply .github/workflows/build-cli.yml:17]: Should still run on regular pushes too though?

Comment (shadaj) [reply .github/workflows/build-cli.yml:17]: Oh true, this makes sense then!

**Pull request merged**